remove relative imports -- fix #855

Corvince opened the pull request:
#855 fails due to sphinx having problem with relative imports, depending on the folder structure. With our current folder structure we can not use relative imports. This PR changes the relative import to absolute ones. 

Further details about the underlying issue can be found at [](https://stackoverflow.com/questions/20251007/sphinx-and-relative-imports-in-python-3)

jackiekazil commented:
What about time? https://github.com/projectmesa/mesa/blob/Corvince-fix-issue-855/mesa/time.py

Corvince commented:
I removed the missing relative imports (time and agent).

Also removed the `# -*- coding: utf-8 -*-` line from all files. It is not needed for python 3. Sorry for mixing up this PR in this respect, but I feel like this is a change that could happen anywhere (and only realized in hindsight that it affected so many files)
